### PR TITLE
Router store

### PR DESCRIPTION
**Pull request by OmerGronich**
A store that exposes the url as an observable and supplies a navigate action

### CODE REVIEW

File: projects/mobx-angular/src/lib/router-store.service.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+    router.events
+      .pipe(filter(event => event instanceof NavigationEnd))
+      .subscribe((e: NavigationEnd) => this.routeListener(e));
+  }
+
+  private routeListener(event: NavigationEnd) {
+    this.url = event.urlAfterRedirects;
+  }
+  navigate(url: string, extras?: NavigationExtras) {
```
[the remark this answers is not in the document]
Comment (OmerGronich) [reply projects/mobx-angular/src/lib/router-store.service.ts]: @adamkleingit 
Hi Adam,
Removed the navigate function from the router store and also added the activated route snapshot as an observable.

Sorry for the long wait on this one

**Review by adamkleingit — APPROVED** [review]
Approved - please resolve conflicts

**Comment by adamkleingit** [comment]
@OmerGronich approved - please resolve conflicts (you can just git checkout --theirs package-lock.json after pulling)

**Comment by OmerGronich** [comment]
> @OmerGronich approved - please resolve conflicts (you can just git checkout --theirs package-lock.json after pulling)

done

**Pull request closed without merging**